### DISCUSSION TITLE
PHP-FPM worker just crashes with no error messages

### DISCUSSION THREAD

**Discussion by tasso85** [laravel/framework#51875] [Q&A]
I am trying to develop an application using Laravel+InertiaJS, and I noticed that I have a problem where sometimes a PHP-FPM worker would just crash without having any error message to check; specifically, I am using Apache 2.4.52 with PHP 8.3.8.

In the FPM log I just see

```
[21-Jun-2024 10:43:51] WARNING: [pool www] child 8414 exited on signal 11 (SIGSEGV) after 543.963041 seconds from start
```

and the web UI shows a modal (I think that is generated by InertiaJS itself) with the message

> Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.
Apache/2.4.52 (Ubuntu) Server at localhost Port 80

in this specific case, I step-debugged my code up to the line where I try to create a new entity:

```
$quote = Quote::create($quote + ["contact_id" => $contact->id]);
```

Here, `$quote` is an array where I initially stored the needed data, in order to later create the model; on that line, it just dies without generating an exception or anything I can check to understand what went wrong.

How do you perform debugging in this cases?

I noticed that if instead I run the app via `php artisan serve` it generates a proper exception; however, I'd like it to work with Apache too, as that is what might be used in production.

**Comment by tasso85** [comment]
Just for curiosity, I tried the Nginx+PHP-FPM combination, the result is the same, only slightly different message in the modal

> 502 Bad Gateway
nginx/1.18.0 (Ubuntu)

So it would seem that the culprit is PHP-FPM itself; however, I wouldn't know how to have it dump a proper log...

Anyway, it seems to me that this is going quite OT for Laravel, unless someone else here has encountered similar problems in the past; if not, maybe I should just close/delete this thread?

**Comment by ghost** [comment]
Hi. Could it be related with the fact that php artisan queue:work without ---once is a long term php process and that some setup from your php fpm shuts it down after a while?

Update another ideea https://stackoverflow.com/a/1564395

**Reply by tasso85** [reply]
I was not talking about the event queue though, but just serving the app.

**Comment by canparam** [comment]
I have the same problem, do you have any solution?
